release/build.gradle

```diff
@@ -33,80 +27,102 @@ description = 'Release module'
+    commandLine 'git', 'sparse-checkout', 'set', "reactive/$projectVersion.family"
+}
+
+/**
+* Update the docs on the cloned website
+*/
+task updateDocumentation( dependsOn: [assembleDocumentation, sparseCheckoutDocumentation] ) {
+    description = "Update the documentation o the cloned static website"
+
+    // copy documentation outputs into target/documentation:
+    // * this is used in building the dist bundles
+    // * it is also used as a base to build a staged directory for documentation upload
 
     doLast {
-        // copy documentation outputs into target/documentation.
-        // 		* this is used in building the dist bundles
-        //		* it is also used as a base to build a staged directory for documentation upload
+        // delete the folders in case some files have been removed
+        delete "${docWebsiteReactiveFolder}/javadocs", "${docWebsiteReactiveFolder}/reference"
 
         // Aggregated JavaDoc
         copy {
-            from "${rootProject.project( 'documentation' ).buildDir}/javadocs"
-            into "${documentationDir}/javadocs"
+            from "${documentationDir}/javadocs"
+            into "${docWebsiteReactiveFolder}/javadocs"
         }
 
         // Reference Documentation
         copy {
-            from "${rootProject.project( 'documentation' ).buildDir}/asciidoc/reference"
-            into "${documentationDir}/reference"
+            from "${documentationDir}/reference"
+            into "${docWebsiteReactiveFolder}/reference"
         }
     }
 }
-assemble.dependsOn assembleDocumentation
 
 /**
- * Configuration to push the documentation on hibernate.org on GitHub
- */
-gitPublish {
-    // where to publish to (repo must exist)
-    repoUri = docPublishRepoUri
-    // branch will be created if it doesn't exist
-    branch = docPublishBranch
-    // where the repository gets cloned
-    repoDir = file("$buildDir/hibernate.org") // defaults to $buildDir/gitPublish
-    // where to fetch from prior to fetching from the remote (i.e. a local repo to save time)
-    // referenceRepoUri = 'file:///home/ddalto/workspace/hibernate.org'
-
-    // what to publish, this is a standard CopySpec
-    contents {
-        from "${documentationDir}"
-        into "reactive/documentation/${projectVersion.family}/"
-    }
-
-    // what to keep in the existing branch (include=keep)
-    preserve {
-        // =================================================================================
-        // WARNING: Keep the include all or everything will be deleted before the copy task
-        // =================================================================================
-        include '**/*'
-        exclude "reactive/documentation/${projectVersion.family}/"
-    }
+* Push documentation changes on the remote repository
+*/
+task stageDocChanges( type: Exec, dependsOn: updateDocumentation ) {
+    workingDir "$project.buildDir/docs-website"
+    commandLine 'git', 'add', '-A', '--sparse', '.'
```

Comment:
Actually you probably needed this only because the `git sparse-checkout set` was incorrect; see https://github.com/hibernate/hibernate-reactive/pull/1196#discussion_r807743691